packages/app/src/components/form/FormItem.vue

```diff
@@ -42,7 +42,7 @@ defineProps({
 }
 .label-inline-block {
   .form-item-label {
-    @apply inline-block;
+    @apply float-left;
```

Comment:
Are you sure it won't break any existing use cases?

Reply:
I don't think so. This class used in two places and I checked them both.